front/lib/api/assistant/configuration.ts

```diff
@@ -890,6 +891,7 @@ export async function createAgentConfiguration(
      */
     const agentConfiguration: AgentConfigurationType = {
       id: agent.id,
+      createdAt: agent.createdAt.getTime(),
```

Comment:
it seems the same thing as "versionCreatedAt" right below, why would we need 2 fields?
Also, if what you needed was the time at which the agent was first created, then this won't be it, you need to fetch version 0.

Reply:
Just did not see it. Closing. Thanks.